spring-batch-core/src/main/java/org/springframework/batch/core/explore/JobExplorer.java

```diff
@@ -95,6 +105,18 @@
 	 */
 	List<JobExecution> getJobExecutions(JobInstance jobInstance);
 
+	/**
+	 * Find the last {@link JobExecution} to have been created for a given
+	 * {@link JobInstance}.
+	 * @param jobInstance the {@link JobInstance}
+	 * @return the last {@link JobExecution} to execute for this instance or
+	 * {@code null} if no job execution is found for the given job instance.
+	 */
```

Comment:
`@since`.  Also what does "last" mean (see previous comments)?

Reply:
Added the `since` tag. The javadoc mentions that it's the last job execution that has been created.